ethstats/ethstats.go

```diff
@@ -796,11 +788,6 @@ func (s *Service) reportStats(conn *connWrapper) error {
 	)
 	// check if backend is a full node
 	if fullBackend, ok := s.backend.(fullNodeBackend); ok {
-		if miningBackend, ok := s.backend.(miningNodeBackend); ok {
-			mining = miningBackend.Miner().Mining()
-			hashrate = int(miningBackend.Miner().Hashrate())
-		}
```

Comment:
Might as well nuke out the variables defined above. I'd also try to remove `Mining` and `Hashrate` from `nodeStats` type definition and see if it choked or not. Hopefully not and then we can get rid of this too.

Reply:
If we can't remove these, we should just blindly set them to some constant.

Reply:
We don't have tests for it, so it doesn't choke :D 
I don't know how to properly test it, without setting up an ethstats instance